### ISSUE TITLE
No docker-ce

### ISSUE THREAD

**Issue by matart15** [digitalocean/supabase-on-do#53]
`packer init .`

```
    digitalocean.supabase: Services to be restarted:
    digitalocean.supabase:  systemctl restart packagekit.service
    digitalocean.supabase:
    digitalocean.supabase: No containers need to be restarted.
    digitalocean.supabase:
    digitalocean.supabase: No user sessions are running outdated binaries.
    digitalocean.supabase:
    digitalocean.supabase: No VM guests are running outdated hypervisor (qemu) binaries on this host.
    digitalocean.supabase: Reading package lists...
==> digitalocean.supabase: E: Could not get lock /var/lib/apt/lists/lock. It is held by process 1795 (apt-get)
==> digitalocean.supabase: E: Unable to lock directory /var/lib/apt/lists/
    digitalocean.supabase: Reading package lists...
    digitalocean.supabase: Building dependency tree...
    digitalocean.supabase: Reading state information...
    digitalocean.supabase: Package docker-ce is not available, but is referred to by another package.
    digitalocean.supabase: This may mean that the package is missing, has been obsoleted, or
    digitalocean.supabase: is only available from another source
    digitalocean.supabase:
==> digitalocean.supabase: E: Package 'docker-ce' has no installation candidate
==> digitalocean.supabase: E: Unable to locate package docker-ce-cli
==> digitalocean.supabase: E: Unable to locate package containerd.io
==> digitalocean.supabase: E: Couldn't find any package by glob 'containerd.io'
==> digitalocean.supabase: E: Couldn't find any package by regex 'containerd.io'
==> digitalocean.supabase: E: Unable to locate package docker-buildx-plugin
==> digitalocean.supabase: E: Unable to locate package docker-compose-plugin
==> digitalocean.supabase: /tmp/script_5994.sh: line 35: docker: command not found
==> digitalocean.supabase: Provisioning step had errors: Running the cleanup provisioner, if present...
==> digitalocean.supabase: Destroying droplet...
==> digitalocean.supabase: Deleting temporary ssh key...
Build 'digitalocean.supabase' errored after 1 minute 20 seconds: Script exited with non-zero exit status: 127. Allowed exit codes are: [0]

==> Wait completed after 1 minute 20 seconds

==> Some builds didn't complete successfully and had errors:
--> digitalocean.supabase: Script exited with non-zero exit status: 127. Allowed exit codes are: [0]
```

<img width="1333" alt="Screenshot 2024-04-01 at 22 09 25" src="https://github.com/digitalocean/supabase-on-do/assets/17027219/2eebc65e-79fd-4df4-83d1-a9a5197eb613">

**Comment by 0reo** [comment]
use ubuntu 22.04, docker-ce is not available on 23.10 yet(assuming that's what you're using based on issue #52 )

**Comment by matart15** [comment]
I see. thank you
